File: src/2769-Find-the-Maximum-Achievable-Number/find_the_maximum_achievable-number.py
class Solution:
    def theMaximumAchievableX(self, num: int, t: int) -> int:
        x: int = 0

        num += t
        x = num + t
        return x

if __name__ == "__main__":
    num: int = 3
    t: int = 7      
    solution = Solution()
    
    print(solution.theMaximumAchievableX(num, t))

File: src/2769-Find-the-Maximum-Achievable-Number/find_the_maximum_achievable-number.cpp
#include <iostream>

class Solution {
public:
    int theMaximumAchievableX(int num, int t) {
        int x = 0;

        num += t;
        x = num + t;
        return x;
    }
};

int main() {
    int num = 3;
    int t = 2;
    
    Solution solution;

    std::cout << solution.theMaximumAchievableX(num, t);
    
    return 0;
}